src/datahike/api.cljc

```diff
@@ -180,20 +213,37 @@
   transact
   dc/transact)
 
+(s/fdef
+  transact!
+  :args (s/cat :conn spec/SConnectionAtom :txs spec/STransactions)
+  :ret #(s/valid? spec/STransactionReport @%))
 (def ^{:arglists '([conn tx-data tx-meta])
        :no-doc    true}
   transact!
   dc/transact!)
 
+(s/fdef
+  load-entities
+  :args (s/cat :conn spec/SConnectionAtom :txs spec/STransactions)
+  :ret #(s/valid? spec/STransactionReport @%)) ; This returns a throwable promise, so we have to dereference it..
```

Comment:
At some point I would drop this function anyway since it will be superceded by the new migration tools.

Reply:
I would still leave the spec in there until the function has actually been dropped.